Lib/fontParts/base/font.py

```diff
@@ -1008,6 +1008,31 @@ def _removeGlyph(self, name, **kwargs):
         layer = self.defaultLayer
         layer.removeGlyph(name)
 
+    def __setitem__(self, name, glyph):
+        """
+        Insert **glyph** into the font. ::
+
+            >>> glyph = font["A"] = otherGlyph
+
+        This will not insert the glyph directly. Rather, a
+        new glyph will be created and the data from **glyph**
+        will be copied to the new glyph. **name** indicates
+        the name that should be assigned to the glyph after
+        insertion. If **name** is not given, the glyph's original
```

Comment:
How can one not give `name` with `__setitem__`?

Reply:
changed in the PR https://github.com/robofab-developers/fontParts/pull/393/files